lib/pbench/test/functional/server/conftest.py

```diff
@@ -62,10 +71,31 @@ def register_test_user(oidc_admin: OIDCAdmin):
         assert response.ok, f"Register failed with {response.json()}"
 
 
+@pytest.fixture(scope="session")
+def register_test_user(oidc_admin: OIDCAdmin):
+    """Create a test user for functional tests."""
+    register_user(oidc_admin, USER)
+
+
+@pytest.fixture(scope="session")
+def register_admintest_user(oidc_admin: OIDCAdmin):
+    """Create a test user matching the configured Pbench admin."""
+    register_user(oidc_admin, ADMIN)
+
+
 @pytest.fixture
 def login_user(server_client: PbenchServerClient, register_test_user):
     """Log in the test user and return the authentication token"""
-    server_client.login(USERNAME, PASSWORD)
+    server_client.login(USER["username"], USER["password"])
+    assert server_client.auth_token
+    yield
+    server_client.auth_token = None
```

Comment:
Would it make sense to combine the guts  of `register_test_user` into this fixture?  (Ditto for `register_admintest_user` and `login_admin`.)

Reply:
Although we currently only register names that we log in, I don't want to force that because I can see reasonable scenarios for registering names we aren't immediately using. I'd rather keep them separate.